vendor/vendor.json

```diff
@@ -3,6 +3,18 @@
 	"ignore": "test",
 	"package": [
 		{
+			"path": "appengine",
+			"revision": ""
```

Comment:
I'll look at that. I wonder why govendor didn't put one in, maybe I was holding it wrong...